src/chopper_layout.cpp

```diff
@@ -162,11 +164,19 @@ size_t determine_best_number_of_technical_bins(chopper::layout::data_store & dat
 
         double const expected_HIBF_query_cost = total_query_cost / total_kmer_count;
 
+        if (t_max_64_memory == -1.0)
+            t_max_64_memory = global_stats.get_total_hibf_memory_size();
+
+        double const relative_memory_size = global_stats.get_total_hibf_memory_size() / t_max_64_memory;
```

Comment:
```suggestion
        double const relative_memory_size = global_stats.get_total_hibf_memory_size() / static_cast<double>(t_max_64_memory);
```

if either operand is a double, the division will be a double division